addon/components/labs-ui/icon-tooltip.hbs

```diff
@@ -0,0 +1,9 @@
+<span class='icon-tooltip'><FaIcon
```

Comment:
Nit: It would be nice to get a comment description of what this component does. I can't tell from this HTML.  I can't easily tell why there needs to be both an <FaIcon> and <EmberTooltip> grouped together. 

Another nit:  IMO it looks nicer and is easier to edit if <FaIcon></FaIcon> is on its own line and tabbed out.

Reply:
I've added notes to `IconTooltipComponent` to clarify what it does and why the whitespace is intentionally suboptimal.